src/main/scala/nl/knaw/dans/easy/pid/ServiceStarter.scala

```diff
@@ -18,13 +18,15 @@ package nl.knaw.dans.easy.pid
 import nl.knaw.dans.easy.pid.microservice.HazelcastService
 import nl.knaw.dans.easy.pid.rest.RestService
 import org.apache.commons.daemon.{Daemon, DaemonContext}
-import org.slf4j.LoggerFactory
+import org.slf4j.{Logger, LoggerFactory}
 
-class PidServiceStarter extends Daemon with SettingsParser {
-  val log = LoggerFactory.getLogger(getClass)
+class ServiceStarter extends Daemon with SettingsParser {
+  var log: Logger = _
```

Comment:
yup, see http://stackoverflow.com/questions/22363950/assign-a-underscore-to-a-variable-what-is-the-underscore-doing for kind of an explanation. Actually I had to look this up. The only reason I did it like this is because IntelliJ told me to do so 😜